CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -474,6 +474,20 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 }
 
 
+- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath {
+    OCKCarePlanActivity *selectedActivity = _events[indexPath.row].firstObject.activity;
+    OCKCarePlanEvent *event = _events[indexPath.row].firstObject;
+
+    [self.store setEndDate:event.date
+               forActivity:selectedActivity
+                completion:^(BOOL success, OCKCarePlanActivity * _Nullable activity, NSError * _Nullable error) {
+                }];
```

Comment:
@umerkhan-apple Are there any other cases when CareKit is showing a notification that something didn't get saved or changed? I thought about it but didn't think throwing an exception was the right thing to do. Would adding `careCardViewController:didDiscontinueActivity:error` be a better way?